altsrc/altsrc.go

```diff
@@ -1,3 +1,3 @@
 package altsrc
 
-//go:generate python ../generate-flag-types altsrc -i ../flag-types.json -o flag_generated.go
```

Comment:
⭐ _(nice to have)_

It'd be nice to have `generate-flags` instead of `fg` here

Reply:
Or `flag-generator`, either one!

Reply:
`flag-generator` it is